src/drom_lib/commandInit.ml

```diff
@@ -0,0 +1,186 @@
+(**************************************************************************)
+(*                                                                        *)
+(*    Copyright 2020 OCamlPro & Origin Labs                               *)
+(*                                                                        *)
+(*  All rights reserved. This file is distributed under the terms of the  *)
+(*  GNU Lesser General Public License version 2.1, with the special       *)
+(*  exception on linking described in the file LICENSE.                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+open Types
+open Ezcmd.TYPES
+open EzCompat
+open EzFile.OP
+
+let cmd_name = "init"
+
+let init_project ~config ~name ~skeleton ~mode ~dir ~args =
+
+  (* all opam files we found *)
+  let opam_files = Opam.get_files () in
+
+  (* try to extract the field `field` using the function `f` from one opam file, if it fails, then use the next opam file. If there's no more opam file, use `default` *)
```

Comment:
The semantics of the new `init` command does not seem obvious to me: it creates only one virtual package, by reading data from `opam` files sorted in a random order ? I think it would be better to either create one package per `opam` or `dune` file, or by only allowing one `opam` file in the project.